CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing
+
+Hyperledger Quilt is a truly open source project. It is a community led effort to provide the components developers need to work with the [Interledger Protocol](https://interledger.org).
+
+The initial versions of Quilt are *_Java_* implementations however any other language implementation is welcomed. Please log an issue if you'd like to contribute using a new lnaguage/stack and we'll get the necessary repo changes made so you can.
+
+First time contributors can either find a module that has not yet been implemented and try that or look for a ["Good First Issue"](https://github.com/hyperledger/quilt/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue") 
+
+## Goals
+
+To use Interledger an application requires one or more of the following components:
+
+  1. Interledger Protocol Codecs
+  1. ILDCP Protocol Codecs
+  1. Bilateral Transfer Protocol Codecs
```

Comment:
Add links to BTP (https://github.com/interledger/rfcs/blob/master/0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md) and Stream (https://github.com/interledger/rfcs/blob/master/0029-stream/0029-stream.md)